server/src/lib/model.ts

```diff
@@ -144,37 +141,42 @@ export default class Model {
       const languages = await this.db.getAllLanguages()
       return languages
     },
-    DAY
+    TimeUnits.DAY,
+    5 * TimeUnits.SECOND,
```

Comment:
I would not touch the 3 minute default in many cases, especially as 5 seconds locking time is too small for some endpoints that need more than 10s to resolve. This way you would just spawn new requests for the same data before the initial request has even finished, leading to wasted calculations.

Reply:
More than 3 min is OK for known problem areas, but less than 3 min is probably not worth the potential wasted resource consumption

Reply:
3 min is the default so you can just omit the parameter here, i.e.
```ts
  async findClipsWithFewVotes(
    client_id: string,
    locale_id: number,
    count: number,
    exemptFromSSRL?: boolean
  ): Promise<DBClip[]> {
    // get cached clips for given language
    const cachedClips: DBClip[] = await lazyCache(
      `new-clips-per-language-${locale_id}`,
      async () => {
        return await this.getClipsToBeValidated(locale_id, 10000)
      },
      TimeUnits.MINUTE
    )()
```